src/custom/components/AccountDetails/index.tsx

```diff
@@ -158,7 +187,13 @@ export default function AccountDetails({
     }
   }, [dispatch, chainId])
   const explorerLabel = chainId && account ? getExplorerLabel(chainId, account, 'address') : undefined
-  const activityTotalCount = (pendingTransactions?.length || 0) + (confirmedTransactions?.length || 0)
+
+  const activities = useMultipleActivityDescriptors({
+    chainId,
+    ids: (pendingTransactions || []).concat(confirmedTransactions || []),
```

Comment:
could and probably should set `pendingTransactions` with a placeholder of [] in the props. also `confirmedTransactions`